sfm/message_consumer/sfm_ui_consumer.py

```diff
@@ -102,13 +104,24 @@ def _on_export_status_message(self):
             log.debug("Updating export with id %s", self.message["id"])
             # Retrieve export model object
             export = Export.objects.get(export_id=self.message["id"])
-
+            # Retrieve user model object
+            user = User.objects.get(username=getattr(export,'user'))
+            # Get reciever's email address
+            reciever_email = getattr(user, 'email')
```

Comment:
Is there a reason you're using getattr instead of just calling the attribute, e.g., `export.user`?

Receiver is spelled "ei" instead of "ie".